product_docs/docs/epas/17/reference/oracle_compatibility_reference/epas_compat_bip_guide/03_built-in_packages/18_dbms_xmldom/getnodename.mdx

```diff
@@ -2,7 +2,7 @@
 title: "GETNODENAME"
 ---
 
-The `GETNODENAME` function provides the name of the node depending on its type.
+The `GETNODENAME` function provides the name of the node, depending on its type.
```

Comment:
I couldn't really parse what "depending on its type" meant. Can you clarify this description? I think it needs elaboration. I wasn't sure if adding the comma was right.

Reply:
You can make a DOMNode out of a DOMDocument, DOMElement, or DOMText. I think this means the `GETNODENAME` fetches the DOMNode name and also underlying type? 

@nidhibhammar and @sravanvelag maybe you can help provide some clarity.

Reply:
We support three types currently: DOMDocument, DOMText and DOMElement. DOMElement will potentially have a valid node name while the other two will have placeholder values '#document' and '#text'. We haven't investigated yet how this function is going to behave for the future types. I am also wondering how this can be elaborated.

Reply:
The behavior for DOMDocument and DOMText will not change (unless some future Redwood version does something different which we plan to inherit). What I meant to convey in my earlier comment was that I am not sure how the description can be elaborated. The first change looks good to me.